src/components/App/App.js

```diff
@@ -86,7 +90,9 @@ class App extends Component {
   }
 
   changePage = (page) => {
```

Comment:
Was actually going to bring this bug up in our next meeting.  Nice fix.  Also makes it so whenever you return to the landing a new title is fetched. I'd say that's a feature